doc/classes/PacketPeer.xml

```diff
@@ -60,6 +60,7 @@
 	</methods>
 	<members>
 		<member name="allow_object_decoding" type="bool" setter="set_allow_object_decoding" getter="is_object_decoding_allowed">
+				If [code]true[/code] the PacketPeer will allow encoding and decoding of object via [method get_var] and [method put_var]. WARNING: Deserialized object can contain code which gets executed. Do not use this option if the serialized object comes from untrusted sources to avoid potential security threats (remote code execution).
```

Comment:
One tab too much for indentation.

Reply:
Maybe use `[b]WARNING:[/b]` to highlight it (and a line break before to make a new paragraph).